data_warehouse/services/aggregate_data_service.py

```diff
@@ -380,3 +385,40 @@ def table_training_feedback(cls, context):
             )
         file_path = TableExportService.export_training_feedback(data)
         return file_path, '培训反馈表.xls'
+
+    @classmethod
+    def table_training_record(cls, context):
+        '''个人培训记录导出'''
+        request = context.get('request')
+        event_name = request.query_params.get('event_name')
```

Comment:
All these parameters can be accessed directly from `context`, need to access them from `request.query_params`